docs/setting-up/server/podman.md

```diff
@@ -0,0 +1,312 @@
+
+## Configuration
+
+### PMM Server
+
+SystemD service passes environment parameters to PMM from `pmm-server.env` file that is located in `~/.config/pmm-server/pmm-server.env`. For more information about container environment variables please check [Docker Environment].
+
+### Customize SystemD service
+
+SystemD service uses some environment variables that could be customized if needed:
+
+```text
+Environment=PMM_PUBLIC_PORT=8443
+Environment=PMM_VOLUME_NAME=%N
+Environment=PMM_TAG=2.29.0
+Environment=PMM_IMAGE=docker.io/percona/pmm-server
+```
+
+Those environment variables could be overridden by defining them in file  `~/.config/pmm-server/env`, for example to override path to custom registry `~/.config/pmm-server/env`:
+
+```sh
+mkdir -p ~/.config/pmm-server/
+cat << "EOF" > ~/.config/pmm-server/env
+PMM_TAG=2.28.0
+PMM_IMAGE=docker.io/percona/pmm-server
+PMM_PUBLIC_PORT=8443
+EOF
+```
+
+!!! caution alert alert-warning "Important"
+    Do modify `PMM_TAG` in `~/.config/pmm-server/env` and update it regularly, as for users there is no way to update it from Percona side and it needs to be done by user.
+
+## Backup
+
+!!! summary alert alert-info "Summary"
+    - Stop PMM server.
+    - Backup the container image.
+    - Backup the data.
+
+---
+
+!!! caution alert alert-warning "Important"
+    Grafana plugins have been moved to the data volume `/srv` since the 2.23.0 version. So if you are upgrading PMM from any version before 2.23.0 and have installed additional plugins then plugins should be installed again after the upgrade.
+    To check used grafana plugins: `podman exec -it pmm-server ls /var/lib/grafana/plugins`
+
+1. Stop PMM server.
+
+    ```sh
+    systemctl --user stop pmm-server
+    ```
+
+2. Backup the container image.
+
+    ```sh
+    podman rename pmm-server pmm-server-backup
+    podman tag pmm-server-backup pmm-server-backup:2.28.0
+    ```
+
+    !!! caution alert alert-warning "Important"
+        Change X.Y.Z (2.28.0) to PMM version you are running, or version of your choice that you could later restore from
+
+
+3. Backup the data.
+
+    ```sh
+    podman volume export pmm-server --output pmm-server-backup.tar
+    ```
+
+    !!! caution alert alert-warning "Important"
+        If you changed default name by `PMM_VOLUME_NAME` environment variable, use that name after `export` instead of `pmm-server` (which is default volume name).
+
+## Upgrade
+
+!!! summary alert alert-info "Summary"
+    - Perform a backup.
+    - Update PMM tag.
+    - Pre-pull image.
+    - Run it.
+
+---
+
+!!! caution alert alert-warning "Important"
+    Downgrades are not possible. To go back to using a previous version you must have created a backup of it before upgrading.
+
+!!! hint alert alert-success "Tip"
+    To see what release you are running, use the *PMM Upgrade* panel on the *Home Dashboard*, or run:
+
+```sh
+podman exec -it pmm-server \
+curl -ku admin:admin https://localhost/v1/version
```

Comment:
Why is this being done with `podman exec`? Just to avoid saying, "install curl"?

Reply:
port mapping was in my mind probably. That command would succeed inside. Outside the container you need alsp mention that port need to be correct.